change/@fluentui-babel-preset-global-context-ebd46a3d-588a-4e6a-9981-389528ddb6f3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27740&commitId=bbf0c6e55ae5763cd8297da1bd80b41dcea3ec61&env=prod)

✅ There was 0 screenshots added, 1 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Size.chromium.png|0| Removed|

</details>


<div id="vrtCommentFluentUI-v0"/>